pages/index.tsx

```diff
@@ -1,23 +1,68 @@
 import type { NextPage } from 'next';
 import Image from 'next/image';
 import Head from 'next/head';
-import styles from '../styles/Home.module.css';
+import styles from '../styles/Home.module.scss';
+import TopCTA from '../components/TopCTA';
+import BottomCTA from '../components/BottomCTA';
+import Footer from '../components/Footer';
 
 const Home: NextPage = () => {
     return (
         <div className={styles.container}>
             <Head>
                 <title>Are we headless yet? | Wagtail</title>
-                <link rel="icon" href="/favicon.ico" />
+                <link rel="icon" href="/images/headless-logo.svg" />
+                <link
+                    rel="preload"
+                    href="/fonts/inter/inter-medium.woff2"
+                    as="font"
+                    crossOrigin=""
+                />
+                <link
+                    rel="preload"
+                    href="/fonts/inter/inter-regular.woff2"
+                    as="font"
+                    crossOrigin=""
+                />
+                <link
+                    rel="preload"
+                    href="/fonts/inter/inter-bold.woff2"
+                    as="font"
+                    crossOrigin=""
+                />
+                <link
+                    rel="preload"
+                    href="/fonts/inter/inter-black.woff2"
+                    as="font"
+                    crossOrigin=""
+                />
             </Head>
 
             <main className={styles.main}>
-                <h1 className={styles.title}>Are we headless yet?</h1>
-
-                <p className={styles.description}>
-                    Get started by editing{' '}
-                    <code className={styles.code}>pages/index.tsx</code>
-                </p>
+                <TopCTA />
+                <div className={styles.hero}>
+                    <div className={styles.logo}>
+                        <Image
+                            src="/images/headless-logo.svg"
+                            alt="Headless Wagtail Logo"
+                            width={140}
+                            height={160}
+                        />
+                    </div>
+                    <div className={styles.hero__body}>
+                        <h1 className={styles.title}>
+                            Are we <br />
+                            <span className={styles.title__emphasis}>
+                                headless
+                            </span>{' '}
+                            yet?
+                        </h1>
+                        <p className={styles.description}>
+                            👍 Yes! But there&apos;s plenty we could improve for
+                            Wagtail&apos;s developers
+                        </p>
+                    </div>
+                </div>
 
                 <div className={styles.grid}>
```

Comment:
If we're ready, we can delete all the boilerplate code here and in `Home.module.scss` that doesn't go into the final design. If it's still useful for testing and page layout feel free to keep it in